packages/shared/charts/src/components/Playground/Playground.tsx

```diff
@@ -53,6 +55,45 @@ const Wrapper = ({ children, renderPanels, renderSidebarHeaderTools }: Playgroun
     );
 };
 
+const SidebarContentTooltip = (): ReactElement => {
+    const [tooltipIsOpen, setTooltipIsOpen] = useState(false);
+    const tooltipTriggerRef = useRef<HTMLButtonElement | null>(null);
+    const tooltipRef = useRef<HTMLDivElement | null>(null);
+
+    useOnClickOutside(tooltipRef, () => setTooltipIsOpen(false));
+
+    return (
+        <button className="info-tooltip" ref={tooltipTriggerRef} onClick={() => setTooltipIsOpen(true)}>
+            <img src={infoIconSvg} className="info-tooltip-icon" alt="Show inline editor information" />
+            {tooltipIsOpen ? (
+                <div
+                    ref={tooltipRef}
+                    className="info-tooltip-info"
+                    style={{
+                        width:
+                            tooltipTriggerRef.current && tooltipTriggerRef.current.parentElement
+                                ? tooltipTriggerRef.current.parentElement.clientWidth * 0.9
+                                : "auto"
+                    }}
+                >
+                    <Alert bootstrapStyle="info" className="info-tooltip-info-content">
+                        <p>
+                            <strong>Changes made in this editor are only for preview purposes.</strong>
+                        </p>
+                        <p>The JSON can be copied and pasted into the widgets properties in the desktop modeler</p>
+                        <p>
+                            Check out the chart options here:{" "}
+                            <a href="https://plot.ly/javascript/reference/" target="_blank" rel="noreferrer">
+                                https://plot.ly/javascript/reference/
+                            </a>
+                        </p>
+                    </Alert>
```

Comment:
We still using Alert?

Reply:
AFAIK yes, but this is also only for the playground editor tooltip and is in line with the previous implementation. Chris will take a look at all the charts anyways before launch, so if he wants to change it he can let us know